cookbooks/convection-box/doc/convection-box.md

```diff
@@ -6,7 +6,9 @@ left and right, and cooled from the top. We will also consider the simplest
 model, the incompressible Boussinesq approximation with constant coefficients
 $\eta,\rho_0,\mathbf g,C_p, k$, for this testcase. Furthermore, we assume that
 the medium expands linearly with temperature. This leads to the following set
-of equations: $$\begin{aligned}
+of equations:
+```{math}
+
```

Comment:
This equation uses multiple & characters to align, for that we need the following here:
```suggestion
\begin{aligned}
```